engine/src/main/java/com/datatorrent/stram/StreamingContainerManager.java

```diff
@@ -2260,6 +2260,29 @@ public void shutdownAllContainers(String message)
     }
   }
 
+  /**
+   * Send STOP_OPERATOR command to all input operator to initiate shutdown of a DAG.
+   *
+   * @param message
+   */
+  public void stopInputOperators(String message)
```

Comment:
engine can shutdown container in various cases, the shutdown field in heartbeat is used to terminate the container in error cases such as blocked operator

This is for user initiated shutdown, also the shutdown field is for whole container, we want to start termination from input operators.